core/src/test/java/org/mskcc/cbio/portal/scripts/TestImportProfileData.java

```diff
@@ -108,6 +113,42 @@ public void testImportMutationsFile() throws Exception {
         validateMutationAminoAcid (geneticProfileId, secondSampleId, 2842, "L113P");
     }
 
+    @Test
+    public void testImportGermlineOnlyFile() throws Exception {
+        /* Mutations file split over two files with same stable id */
+        String[] args = {
+                "--data","src/test/resources/germlineOnlyMutationsData/data_mutations_extended.txt",
+                "--meta","src/test/resources/germlineOnlyMutationsData/meta_mutations_extended.txt",
+                "--loadMode", "bulkLoad"
+        };
+        ImportProfileData runner = new ImportProfileData(args);
+        runner.run();
+        String studyStableId = "study_tcga_pub";
+        CancerStudy study = DaoCancerStudy.getCancerStudyByStableId(studyStableId);
+        studyId = study.getInternalId();
+        int sampleId = DaoSample.getSampleByCancerStudyAndSampleId(studyId, "TCGA-AA-3664-01").getInternalId();
+        GeneticProfile geneticProfile = DaoGeneticProfile.getGeneticProfileByStableId(studyStableId + "_breast_mutations");
+        geneticProfileId = geneticProfile.getGeneticProfileId();
+
+        // assume clinical data for MUTATION_COUNT was created
+        ClinicalAttribute clinicalAttribute = DaoClinicalAttributeMeta.getDatum("MUTATION_COUNT", studyId);
+        assertNotNull(clinicalAttribute);
+
+        // assume a MUTATION_COUNT record has been added for one sample and the count is zero
+        List<ClinicalData> clinicalData = DaoClinicalData.getSampleData(study.getInternalId(), new ArrayList<String>(Arrays.asList("TCGA-AA-3664-01")), clinicalAttribute);
+        assert(clinicalData.size() == 1);
+        assertEquals("0", clinicalData.get(0).getAttrVal());
+
+        // check if the three germline mutations have been inserted
+        validateMutationAminoAcid (geneticProfileId, sampleId, 64581, "T209A");
+        validateMutationAminoAcid (geneticProfileId, sampleId, 50839, "G78S");
+        validateMutationAminoAcid (geneticProfileId, sampleId, 2842, "L113P");
+
+        // remove profile at the end
+        DaoGeneticProfile.deleteGeneticProfile(geneticProfile);
+        assertNull(DaoGeneticProfile.getGeneticProfileByStableId(studyStableId + "_breast_mutations"));
```

Comment:
since this is not a test of the DaoGeneticProfile.deleteGeneticProfile() function, we probably do not need an assert statement here.

Reply:
i've addressed this in https://github.com/cBioPortal/cbioportal/pull/4951. These lines can be removed after merging that